Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -136,6 +146,12 @@ public async Task RunAsync()
         {
             await ((DlnaManager)_dlnaManager).InitProfilesAsync().ConfigureAwait(false);
 
+            if (_disabled)
```

Comment:
Shouldn't this check occur before `InitProfilesAsync`?

Reply:
The profiles are used elsewhere in the system in regard to streaming.
That's the only part of the dlna system that i haven't been able to migrate to a plugin.